script.js

```diff
@@ -1 +1,38 @@
-// this is script file
+import { sanitizaInput, clearInputField } from "./utility.js";
+
+const $todoInput = document.getElementById("input-field");
+const $addButton = document.getElementById("add-button");
+const $todoList = document.getElementById("todo-list");
```

Comment:
Move this and associated event listener function to another file.
e.g:
```suggestion
// elements.js
export const $todoInput = document.getElementById("input-field");
export const $addButton = document.getElementById("add-button");
export const $todoList = document.getElementById("todo-list");

$addButton.addEventListener("click", addTodoHandler);
...

// index.js
import { $todoInput } from "./elements.js"
```